pkg/fanal/artifact/local/fs_test.go

```diff
@@ -415,10 +432,6 @@ func TestTerraformMisconfigurationScan(t *testing.T) {
 			want: types.ArtifactReference{
 				Name: "testdata/misconfig/terraform/single-failure/src",
 				Type: types.ArtifactFilesystem,
-				ID:   "sha256:b3ae72efb468a0e17551fa4067c1f9d9dff9a1e520b9f8191f48829ab6e8356d",
-				BlobIDs: []string{
-					"sha256:b3ae72efb468a0e17551fa4067c1f9d9dff9a1e520b9f8191f48829ab6e8356d",
-				},
```

Comment:
note: I've removed these assertions as the key is calculated over everything (including the file paths in the blob), which results in the key being different on different OS's (unix vs windows).